feature/system/logging/console_vty_file/tests/README.md

```diff
@@ -0,0 +1,81 @@
+# TR-6.2 Local logging destinations
+
+## Sumary
+Verify if required OC configuration is accepted by DUT and exposed via gNMI get
+
+## Prodecure
+
+### Configuration
+
+### TC1 - console logging configuration
+
+1. configure and enable consloe logging with:
+    - 2 selectors:
+      - facility = "Local7", severity = "informational"
+      - facility = "Local6", severity = "alarm"
+      - facility = "Local5", severity = "critical"
+    - remote facility rewrites:
+      - "local7" --> "local2"
+      - "local6" --> "local3"
+  Note: Facility "Local5" is not rewritten
+2. Read consloe logging configuration and compate with pushed configuration.\
+   Note: two selectors must be presented.
+3. disable consloe logging while keeping 2 selectors configured:
+4. Read consloe logging configuration and compate with pushed configuration.\
+   Note: two selectors must be presented.
+
+### [TODO] TC2 - VTY logging configuration
+The vty represents here terminal session - ssh, telnet.
+> NOTE: This is NOT yet modeled in OpenConfig
+
+1. configure and enable vty logging with 2 selectors:
+    - facility = "Local7", severity = "informational"
+    - facility = "Local5", severity = "alarm"
+2. Read vty logging configuration and compate with intended configuration.\
+   Note: two selectors must be presented.
+3. disable vty logging while keeping 2 selectors configured:
+4. Read vty logging configuration and compate with intended configuration.\
+   Note: two selectors must be presented.
+
+### [TODO] TC2 - buffer logging configuration
+> NOTE: This is NOT yet modeled in OpenConfig
```

Comment:
This is true.  @Swetha-haridasula can you open an issue for yourself to track adding "buffer" as a logging destination in OpenConfig?